eZ/Publish/Core/REST/Server/Controller/Role.php

```diff
@@ -115,6 +126,32 @@ public function createRole(Request $request)
     }
 
     /**
+     * Creates a new RoleDraft for an existing Role.
+     *
+     * @since 6.1
```

Comment:
Wouldn't it be 6.2 ? The current version is already 6.1. And if it is not merged before the freeze, it would even be 6.3.